app/src/main/java/com/fish/monsters/features/history/ui/components/HistoryItem.kt

```diff
@@ -0,0 +1,164 @@
+package com.fish.monsters.features.history.ui.components
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.AccessTime
+import androidx.compose.material.icons.filled.CalendarMonth
+import androidx.compose.material.icons.filled.SportsScore
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.drawWithContent
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.DpSize
+import androidx.compose.ui.unit.dp
+import com.fish.monsters.R
+import com.fish.monsters.common.extensions.drawNeonStroke
+import com.fish.monsters.common.extensions.isPreview
+import com.fish.monsters.common.utils.settings.SettingsManager
+import com.fish.monsters.common.views.PreviewContainer
+import com.fish.monsters.common.views.buttons.OutlinedFishButton
+import com.fish.monsters.core.database.entities.Contest
+import com.fish.monsters.core.database.entities.contest.Award
+import com.fish.monsters.core.theme.DarkPrimaryColor
+import com.fish.monsters.core.database.entities.contest.DifficultyLevel
+import com.fish.monsters.core.database.entities.contest.Duration
+import com.fish.monsters.core.database.entities.contest.GameLocation
+import org.koin.compose.koinInject
+
+@Composable
+fun HistoryItem(
+    contestInfo: Contest,
+    neonStyle: Boolean = if (isPreview()) false else koinInject<SettingsManager>().state.value.neonStyles,
+) {
+    OutlinedFishButton(
+        modifier = Modifier
+            .fillMaxWidth()
+            .height(66.dp)
+            .drawWithContent {
+                drawContent()
+                if (neonStyle)
+                    drawNeonStroke(DpSize(12.dp, 31.dp), DarkPrimaryColor)
+            },
+        onClick = { println(contestInfo) },
+        indentationSize = DpSize(12.dp, 31.dp),
+        contentPadding = PaddingValues(13.dp, 14.dp),
+    ) {
+        Row(Modifier.fillMaxSize()) {
+            Column(Modifier.width(200.dp)) {
```

Comment:
nie powinniśmy statycznie określać takich szerokości, wielkość ekranu może być różna, lepiej to zrobić dynamicznie, a jak to poniżej